src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Imaging/PropVariant.cs

```diff
@@ -364,7 +364,12 @@ internal void Init(object value)
                 else if (value is char)
                 {
                     varType = (ushort)VarEnum.VT_LPSTR;
+
+#if NET
                     pszVal = Marshal.StringToCoTaskMemAnsi(new String(stackalloc char[] { (char)value }));
```

Comment:
Does stackalloc char[] -> Span work in .NET Framework Eric, and if so, can this just be that followed by ToString on the resulting Span?